includes/islandora_book_batch.inc

```diff
@@ -475,9 +475,11 @@ class IslandoraBookPageBatchObject extends IslandoraFlatBatchObject {
     islandora_paged_content_set_relationship($rels_ext, FEDORA_RELS_EXT_URI, 'isMemberOf', $this->parentId);
     // Add content model relationship.
     $this->models = 'islandora:pageCModel';
-    if (isset($this->preprocessorParameters['generate_ocr'])) {
+    if (isset($this->preprocessorParameters['generate_ocr']) || isset($this->preprocessorParameters['generate_hocr'])) {
```

Comment:
This one differs from https://github.com/Islandora/islandora_newspaper_batch/pull/21/files/dc6a0e2017cd3192f640b6ec7f65e58b080040de#diff-bb0eb85e45687085a3b6815a5c75a4a2R494 Any reason? The newspaper one says "if one is set, both are set". Does that apply here also?

Reply:
Ah yes, this could be simplified. The reasoning here is that the existence of these parameters is dependent on the existence of the `islandora_ocr` module; if `islandora_ocr` is installed, you'll have a value for both of these. If it's not, neither of them will be set.

Speaking of which, it looks like the drush batch setter doesn't contain any such logic and passes those values into the RELS-EXT no matter what; I'm gonna patch that